lib/mumukit/platform/uri.rb

```diff
@@ -8,11 +8,11 @@ def subdominate(subdomain)
     rebuild(host: new_host)
   end
 
-  def subroute(route)
-    if path.start_with? '/'
-      new_path = "/#{route}#{path}"
+  def tenantize(route)
```

Comment:
I'm not happy with the name :confused:

Reply:
:thought_balloon: From my POV `tenantize` is just another case of "subrouting". Perhaps `subroute(route, append: true)` 

Another option my by just calling it `append`: `uri.append(route)`. However with that name I'd strongly suggest handling query params and fragments.

Reply:
We've discussed this and are happy with the name after all! :laughing: